aws/v12.4.0/announcement.md

```diff
@@ -0,0 +1 @@
+**Giant Swarm release v12.4.0 for AWS is available.** This release upgrades updates Cert-Manager to `2.3.0` that brings new patch version `v1.0.2` of the upstream project. Further details can be found in the [release notes](https://github.com/giantswarm/releases/tree/master/aws/v12.4.0).
```

Comment:
I also saw that external-dns was also mentioned but it's not in the release notes or the announcement. Are the changes not relevant?

Reply:
minor bugs but let me add it